store/postgres/src/relational.rs

```diff
@@ -162,6 +162,18 @@ impl fmt::Display for SqlName {
     }
 }
 
+impl Borrow<str> for SqlName {
+    fn borrow(&self) -> &str {
+        self.as_str()
+    }
+}
+
```

Comment:
We don't need to implement `Borrow<str>` for `SqlName`, only for `&SqlName`.

```suggestion
```

Reply:
Nice catch. Taking that out.